### PR TITLE
Enable selection of annotation scanners (#1370) via maven plugin config.

### PR DESCRIPTION
**Pull request by Olard**
Hi,

this PR adds the capability to configure which annotation scanners should be executed (as noted in #1370).

### CODE REVIEW

**Review by MikeEdgar — APPROVED** [review]
This looks good to me. @phillip-kruger wdyt?

File: tools/maven-plugin/README.adoc [hunk 1]
```diff
@@ -66,6 +66,7 @@ E.g. for includeDependenciesScopes could be configured as:
 - `skip` (boolean, default: false) - Skip execution of the plugin.
 - `encoding` (String) - Encoding of output OpenAPI files.
 - `outputFileTypeFilter` (String, default: ALL) - Set this to `YAML` in order to let the generation process produce just *.yaml output, `JSON` to obtain *.json files only. The default is `ALL`, which will generate both file types.
+- `scanners` (List<String>) - To set the names of the AnnotationScanner implementations that should be executed.
```
Comment (MikeEdgar) [inline tools/maven-plugin/README.adoc:69]:
```suggestion
- `scanners` (List<String>) - Names of the AnnotationScanner implementations that should be executed. If omitted or an empty list, all scanners will be executed. The names of the scanners available by default are `JAX-RS`, `Spring`, and `Vert.x`.
```

**Review by MikeEdgar — CHANGES_REQUESTED** [review]
@Olard , minor update to the docs suggested. Please take a look.

**Comment by Olard** [comment]
> @Olard , minor update to the docs suggested. Please take a look.

Yes, seems more informative that way : )

**Review by MikeEdgar — APPROVED** [review]
Thanks for the PR

**Pull request merged**